packages/devextreme/scss/widgets/generic/menu/_index.scss

```diff
@@ -46,6 +46,10 @@
   }
 
   .dx-menu-item-has-text {
+    &.dx-state-focused {
```

Comment:
Why do u do it only for `.dx-menu-item-has-text`? It does not work now if item contains an icon icon